stories/index.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
 import { storiesOf } from '@storybook/react'
 import { action } from '@storybook/addon-actions'
-import Avatar from 'material-ui/Avatar'
-import Chip from 'material-ui/Chip'
-// import AutoComplete from 'material-ui/AutoComplete'
-import { green } from 'material-ui/colors'
+import Avatar from '@material-ui/core/Avatar'
+import Chip from '@material-ui/core/Chip'
+// import AutoComplete from '@material-ui/core/AutoComplete'
```

Comment:
I think we can remove this line. `AutoComplete` will never be in `@material-ui/core` :cry:

Reply:
@leMaik nm - you already took care of that :-)

Reply:
Indeed, I just _couldn't_ wait. :smile: